packages/safe-apps-sdk/README.md

```diff
@@ -114,10 +114,12 @@ Once you received safe transaction hash, you might want to get the status of the
 const tx = await sdk.txs.getBySafeTxHash(safeTxHash);
 ```
 
-It will return the following structure https://github.com/gnosis/safe-apps-sdk/blob/development/src/types.ts#L182 or throw an error if the backend hasn't synced the transaction yet
+It will return a [TxServiceModel](src/types.ts#L164-L194) structure or throw an error if the backend hasn't synced the transaction yet
 
 ## RPC Calls
 
+The SDK exposes Ethereum's [JSON-RPC methods](https://eth.wiki/json-rpc/API).
```

Comment:
I'm not sure whether we should mention that we only expose read methods, not the whole API 🤔

Reply:
I'll adjust it 👌